src/main/frontend/fs/capacitor_fs.cljs

```diff
@@ -174,7 +174,9 @@
                    (string/replace "//" "/")))]
     (if (mobile-util/native-ios?)
       (js/encodeURI (js/decodeURI path))
-      path)))
+      (if (string/starts-with? path "file://")
```

Comment:
The whole `get-file-path` function seems buggy.
🤔

Reply:
- above `(string/replace "///" "/")` breaks absolute file path like `file:///var/...`

Reply:
It's designed on purpose. On iOS, we need a final path (`file:/private/var/mobile/Library/Mobile Documents/iCloud~com~logseq~logseq/Documents/`) to pass.